SageMaker-Installation-Instructions.md

```diff
@@ -0,0 +1,44 @@
+### Ensuring permissions for your SageMaker execution role
+
+Make sure to add the correct permissions to the role your SageMaker Studio runs with. A good place to start is by adding the `AwsGlueDataBrewFullAccessPolicy` managed policy to your selected SageMaker execution role, and then adding inline permissions allowing `s3:GetObject` and `s3:PutObject` for the bucket in which you would like to operate. This grants  baseline permissions to use AWS Glue DataBrew console features. Refer to DataBrew developer guide on [Setting up IAM permissions](https://docs.aws.amazon.com/databrew/latest/dg/setting-up-iam.html).
```

Comment:
Sorry may have missed this earlier - I think we should avoid recommending the use of inline permissions. The best practice for IAM is to use a customer managed policy in order to provide visibility. We can link them directly to this page in the documentation for setting up their IAM permissions for S3 access: https://docs.aws.amazon.com/databrew/latest/dg/iam-policy-to-use-s3-for-data-resource-role.html